kcidb/orm/data.py

```diff
@@ -0,0 +1,478 @@
+"""
+The kcidb.orm.data module defines the data structures and constants used
+by the KCIDB object-relational mapping system to organize Kernel CI report
+data into objects.
```

Comment:
Let's not put the module name into its description, so we don't forget to update it if we ever rename it. Besides, it's redundant. You know that it's a module, and which module it is when you're reading this. Otherwise this is a pretty good description, thanks! But let's start it with just "The data structures and constants ..." directly.